### PR TITLE
Create gravy.json

### PR DESCRIPTION
**Pull request by ThatGravyBoat**

### CODE REVIEW

**Review by MaskDuck — APPROVED** [review]
wait not

File: domains/gravy.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "thatgravyboat",
+    "twitter": "thatgravyboat",
+    "discord": "thatgravyboat#0001"
```
Comment (MaskDuck) [inline domains/gravy.json:5]: email pls? if not wanna leak, leave an empty string like `{"email": ""}`

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/gravy.json (hunk 1 above)
Comment (ThatGravyBoat) [reply domains/gravy.json]: Fixed

**Review by MaskDuck — APPROVED** [review]
looks not sus.
btw do you wanna join our [discord server](https://discord.gg/dJnGjeWMHx)?

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**